Animation/Animation.cpp

```diff
@@ -22,7 +22,7 @@ namespace {
 
```

Comment:
Also, not a fun of using global and alternating then through class method. I think g_step_ms and g_fps should be a members of Animation class.